litgpt/config.py

```diff
@@ -99,7 +108,7 @@ def __post_init__(self):
         self.rope_n_elem = int(self.rotary_percentage * self.head_size)
 
         if self.sliding_window_size is not None:
-            self.sliding_window_layer_placing = (
+            self.sliding_window_layer_period = (
```

Comment:
To be honest, I'm not sure that it's an issue, since python allows it :) 
But, if it looks confusing for someone who looks at the code in the very first time, we can rename it.

Perhaps we should use `stride` instead of `period`?
```suggestion
            self.sliding_window_layer_stride = (
```

Reply:
My IDE certainly complains